src/EnergyPlus/Autosizing/HeatingCapacitySizing.hh

```diff
@@ -49,7 +49,6 @@
 #define HeatingCapacitySizing_hh_INCLUDED
 
 #include <EnergyPlus/Autosizing/BaseSizerWithScalableInputs.hh>
-#include <EnergyPlus/DataHVACGlobals.hh>
```

Comment:
Duplicate include seemed to cause Intellisense to report "incomplete type not allowed" (see issue description)